### DISCUSSION TITLE
[Feature] show password in password-entry fields

### DISCUSSION THREAD

**Discussion by jonhnet** [immich-app/immich#5855] [Feature Request]
### The feature

My wife is having frustration logging into Immich and setting and resetting her password. Part of that frustration is trying to see if the password she typed into the browser or mashed into her iPhone is what she meant to type. She's not worried about shoulder surfers; she would benefit greatly from the little "eye" button that lets her see what she's typing in the password field.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Jan108** [comment]
It is implemented by #6918 and #7368.
